webserver/templates/user/profile.html

```diff
@@ -31,11 +31,21 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
             </thead>
             <tbody>
             {% for listen in listens %}
+
+              {% if listen.playing_now %}
+              <tr id="playing_now">
+                <td>{{ listen.track_metadata.artist_name }}</td>
+                <td>{{ listen.track_metadata.track_name }}</td>
+                <td><span class="glyphicon glyphicon-play" aria-hidden="true"></span> Playing now</td>
+              </tr>
```

Comment:
Doesn't look like you did. :) Code inside `if`s, `for`s, and other blocks should be indented.